std_msgs/Header header
int32 pos_x
int32 pos_y
int32 pos_z
int32 vx
int32 vy
int32 vz
int32 pitch
int32 roll
int32 yaw
uint8 quality
uint8 reloc

